src/blocks/pacman.rs

```diff
@@ -192,8 +192,10 @@ impl ConfigBlock for Pacman {
 }
 
 fn run_command(var: &str) -> Result<()> {
-    Command::new("sh")
-        .args(&["-c", var])
+    Command::new("fakeroot")
+        .args(&["--", var])
+        .stdout(Stdio::null())
+        .stderr(Stdio::null())
```

Comment:
Maybe leave `stderr` piped? If something is printed to `stderr` it doesn't break the bar, but might (?) be useful for debugging.

Reply:
sounds logical. Will do.